packages/main/src/ListItem.ts

```diff
@@ -337,26 +337,6 @@ abstract class ListItem extends ListItemBase {
 		this.fireItemPress(e);
 	}
 
-	/*
-	 * Called when selection components in Single (ui5-radio-button)
-	 * and Multi (ui5-checkbox) selection modes are used.
-	 */
-	onMultiSelectionComponentPress(e: MouseEvent) {
-		if (this.isInactive) {
-			return;
-		}
-
-		this.fireEvent<SelectionRequestEventDetail>("_selection-requested", { item: this, selected: (e.target as HTMLInputElement).checked, selectionComponentPressed: true }); // Switch HTMLInputElement to CheckBox when ui5-check is migrated to TypeScript
-	}
-
-	onSingleSelectionComponentPress(e: MouseEvent) {
-		if (this.isInactive) {
-			return;
-		}
-
-		this.fireEvent<SelectionRequestEventDetail>("_selection-requested", { item: this, selected: !(e.target as ListItemBase).selected, selectionComponentPressed: true });
```

Comment:
The idea for the event, fired from the radio and checkbox, and from there from the LsitItem to the List,
was to have this `selectionComponentPressed` parameter in the event.detail - indicating if the user clicked on the selection component or somewhere else. I found the PR that introduces this, but could not find any usage or issue linked with it.

So, I don't find a reason to keep them. But the change is not complete. You need to remove `selectionComponentPressed ` from the List as well, this is example for one of the places it's declared (but there are more)
```js
@event("selection-change", {
	detail: {
		selectedItems: { type: Array },
		previouslySelectedItems: { type: Array },
		targetItem: { type: HTMLElement }, // protected, holds the event target item
		selectionComponentPressed: { type: Boolean }, // protected, indicates if the user used the selection components to change the selection
	},
})
```

Reply:
Actually, I found a usage in the MultiComboBox:

<img width="1033" alt="Screenshot 2023-02-27 at 12 08 17" src="https://user-images.githubusercontent.com/15702139/221534944-69835f28-3b2b-4343-b5f4-63e4ecbc1589.png">

Reply:
I noticed that the MultiComboBox tests are unstable with the change. I'll check if the event in question can be deleted or I'll need a workaround.